technical/XSD_EXTENSIONS.md

```diff
@@ -0,0 +1,40 @@
+<!--
+SPDX-FileCopyrightText: 2021 Alliander N.V.
+
+SPDX-License-Identifier: CC-BY-4.0
+-->
+
+## XSD Extensions
+For generating the SCL classes, we use the official IEC SCL XSD schemas. These are stored in the `scl-extension` module of the [CoMPAS Core repository](https://github.com/com-pas/compas-core). An other option was using the [CoMPAS IEC XSD repository](https://github.com/com-pas/compas-scl-xsd), but we want to keep that repository dedicated to the IEC XSD schemas.
```

Comment:
Don't understand the first sentence, the official IEC SCL XSD schemas are generated in 'scl-extension'.